tests/step-061.sh

```diff
@@ -12,7 +12,7 @@ mkdir -p /tmp/shared/first-shared.git/.githooks/pre-commit &&
     (cd /tmp/shared/second-shared.git && git init && git add . && git commit -m 'Testing') ||
     exit 1
 
-git config --global githooks.shared '/tmp/shared/first-shared.git' || exit 1
+git config --global githooks.shared 'file:///tmp/shared/first-shared.git' || exit 1
```

Comment:
Did this change intentionally? Absolute paths should still be OK for this config.

Reply:
Jup I can revert this. Thx

Reply:
Nope, cant change that its needed for the logic later, it needs to be a cloned repo here.